test/test_acyclic.jl

```diff
@@ -3,8 +3,9 @@ using LightGraphs
 using Test
 
 using Random
-Random.seed!(123)
+Random.seed!(45)
```

Comment:
Why changing the seed?

Reply:
Oh, I was trying out different inputs for the algorithm by changing the seed at every run. Forgot to set it back to the original value.